ci-operator/jobs/openshift/release/openshift-release-release-4.1-periodics.yaml

```diff
@@ -697,34 +697,38 @@ periodics:
       env:
       - name: CONFIG_SPEC
         value: |
-          tag_specification:
-            name: "4.1"
-            namespace: ocp
           resources:
             '*':
               limits:
                 memory: 4Gi
               requests:
                 cpu: 100m
                 memory: 200Mi
+          tag_specification:
+            name: "4.1"
+            namespace: ocp
           tests:
           - as: e2e-aws
             commands: TEST_SUITE=openshift/conformance/parallel run-tests
             openshift_installer:
-              cluster_profile: "aws"
+              cluster_profile: aws
           - as: e2e-aws-serial
             commands: TEST_SUITE=openshift/conformance/serial run-tests
             openshift_installer:
-              cluster_profile: "aws"
+              cluster_profile: aws
           - as: e2e-aws-upgrade
             commands: TEST_SUITE=all run-upgrade-tests
             openshift_installer:
-              cluster_profile: "aws"
+              cluster_profile: aws
               upgrade: true
           - as: launch-aws
             commands: sleep 9000 & wait
             openshift_installer:
-              cluster_profile: "aws"
+              cluster_profile: aws
+          zz_generated_metadata:
+            branch: ""
+            org: ""
+            repo: ""
```

Comment:
Can't the empty metadata break anything?

Reply:
I don't think so, no. The operator itself doesn't use it.